.github/workflows/alert-community.yml

```diff
@@ -0,0 +1,40 @@
+name: "[service] Alert Community"
+
+on:
+  pull_request:
+   types: [opened] 
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/alert-community
+        with:
+          files: "docs/*/*"
+          reviewers: "@prince-chrismc"
+        
+      - uses: ./.github/actions/alert-community
+        with:
+          files: ".github/*/*"
+          reviewers: "@ericLemanissier @prince-chrismc"
+        
+      - uses: ./.github/actions/alert-community
+        with:
+          files: "linter/*/*"
+          reviewers: "@ericLemanissier @prince-chrismc"
+          
+      - uses: ./.github/actions/alert-community
+        with:
+          files: "recipes/aaf/*/*"
+          reviewers: "@MartinDelille"
```

Comment:
@MartinDelille You previously expressed interest in following this recipe, let me know if this is okay to add you too :)

Reply:
Yes!

Reply:
I'll be happy to have notification on more recipes (like ffmpeg). Can I specify a list here ?